packages/api/src/schema/schema.yaml

```diff
@@ -811,6 +813,354 @@ components:
+              example: 1920
+            height:
+              type: number
+              description: height of the profile
+              example: 1080
+            frameRate:
+              type: number
+              description: Frame rate of the profile
+              example: 30
+            id:
+              type: string
+              description: ID of the profile
+              example: ABC123
+            duration:
+              type: number
+              description: Duration of the profile in seconds
+              example: 23.8238
+            bitrate:
+              type: number
+              description: Bitrate of the profile in bits per second
+              example: 1000000
+        playbackUrl:
+          type: string
+          description: Playback url of the playlist manifest
+          example: https://cdn.livepeer.com/ABC123/playlist.m3u8
+        status:
+          type: string
+          description: Status of the asset
+          example: ready
+        originTaskId:
+          type: string
+          description: Task ID of the task that created the asset
+          example: 1-23-123-123-1123-123
+        relatedTasks:
+          type: array
+          description: List of task IDs associated with the asset
+          items:
+            $ref: "#/components/schemas/task"
+        sourceAssetId:
+          type: string
+          description:
+            ID of the source asset (root) - If missing, this is a root asset
+          example: 123123123123
+        nft:
+          type: array
+          description: List of NFTs minted with the asset as a source
+          items:
+            type: object
+            properties:
+              chain:
+                type: string
+                description: Chain of the NFT
+                example: eth
+              contract:
+                type: string
+                description: Contract of the NFT
+                example: 0x57f1887a8bf19b14fc0d912b9b2acc9af147ea85
+              token:
+                type: number
+                description: Token id of the NFT
+                example: 123
+
+    task:
+      type: object
+      table: task
+      properties:
+        id:
+          type: string
+          description: Task ID
+          example: 1-23-123-123-1123-123
+        kind:
+          type: string
+          description: Type of the task
+          example: import
+        status:
+          type: string
+          description: Status of the task
+          example: ready
+        createdAt:
+          type: string
+          description: Timestamp (in milliseconds) at which task was created
+          example: 2022-01-01T02:20:00Z
+        updatedAt:
+          type: string
+          description: Timestamp (in milliseconds) at which task was updated
+          example: 2022-01-01T02:20:00Z
+        params:
+          type: object
+          description: Parameters of the task
+          properties:
+            import:
```

Comment:
Should we have the params further broken down into `Import`, `Export` and `Transcode` groups, or the `kind` parameter is enough and the params could be on the same level in the object?

Reply:
I think the params should be broken down between each `type`, no matter if we have the top-level `params` field or not.

Initially, I had thought about the `import`, `export` and transcode` fields directly on the root of the `task` object, but then they'd probably need a prefix or sufix like `importParams` or `paramsImport`, which looks a bit cumbersome. So I did like having the `params` on the root and then the field with the task type as a key! Looks neat. 

We could do with the params fields directly on the root though, but I do have a strong preference for the params for each separate task contained within their own sub-field (`import`, `export`, etc)